### DISCUSSION TITLE
How to get the data back from item

### DISCUSSION THREAD

**Discussion by hooramin** [fkhadra/react-contexify#233] [Q&A]
Hi there,
I am new to react and Javascript in general.
I want to use this lib. I can make the menu appear based on the guide. But I am struggling to get the data back. I am lost.
I need to know what was the id or name of my Div that right clicked happened on it, So I can pass it to the function I need. 

But I don't know what clicked. I don't get the name or id or value of that element that received the right click in the console, 
I tried to print props, event, id and none had useful data.
Can anyone guide me or send me a link to learn more about this?

**Comment by yulei-chen** [comment]
I guess prop `data` of menu item is what you need
see https://fkhadra.github.io/react-contexify/disable-or-hide
